src/app/zap-templates/zcl/data-model/chip/joint-fabric-datastore-cluster.xml

```diff
@@ -0,0 +1,370 @@
+      <arg id="0" name="EndpointID" type="endpoint_no"/>
+      <arg id="1" name="NodeID" type="node_id"/>
+      <arg id="2" name="FriendlyName" type="char_string" length="32"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x0E" name="AddGroupIDToEndpointForNode" isFabricScoped="true">
+      <description>This command SHALL be used to add a Group ID to an endpoint for a node in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="NodeID" type="node_id"/>
+      <arg id="1" name="EndpointID" type="endpoint_no"/>
+      <arg id="2" name="GroupID" type="group_id"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x0F" name="RemoveGroupIDFromEndpointForNode" isFabricScoped="true">
+      <description>This command SHALL be used to remove a Group ID from an endpoint for a node in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="NodeID" type="node_id"/>
+      <arg id="1" name="EndpointID" type="endpoint_no"/>
+      <arg id="2" name="GroupID" type="group_id"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x10" name="AddBindingToEndpointForNode" isFabricScoped="true">
+      <description>This command SHALL be used to add a binding to an endpoint for a node in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="NodeID" type="node_id"/>
+      <arg id="1" name="EndpointID" type="endpoint_no"/>
+      <arg id="2" name="Binding" type="DatastoreBindingTargetStruct"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x11" name="RemoveBindingFromEndpointForNode" isFabricScoped="true">
+      <description>This command SHALL be used to remove a binding from an endpoint for a node in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="ListID" type="int16u"/>
+      <arg id="1" name="EndpointID" type="endpoint_no"/>
+      <arg id="2" name="NodeID" type="node_id"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x12" name="AddACLToNode" isFabricScoped="true">
+      <description>This command SHALL be used to add an ACL to a node in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="NodeID" type="node_id"/>
+      <arg id="1" name="ACLEntry" type="DatastoreAccessControlEntryStruct"/>
+      <access op="invoke" privilege="administer"/>
+    </command>
+
+    <command source="client" code="0x13" name="RemoveACLFromNode" isFabricScoped="true">
+      <description>This command SHALL be used to remove an ACL from a node in the Joint Fabric Datastore Cluster.</description>
+      <arg id="0" name="ListID" type="int16u"/>
```

Comment:
Spec issue: same problem with ListID as for bindings.

Reply:
Tracked by https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11620

Reply:
The spec change will be tracked using [issue #11620](https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/11620). Once the spec update is complete, a follow-up SDK PR can be submitted to align it with the updated specification.